tests/Unit/Domain/Creators/Test_AlignedLattice.cpp

```diff
@@ -89,17 +89,19 @@ template <size_t VolumeDim>
 auto make_domain_creator(const std::string& opt_string,
                          const bool use_boundary_condition) {
   if (use_boundary_condition) {
-    return TestHelpers::test_factory_creation<
-        DomainCreator<VolumeDim>, domain::OptionTags::DomainCreator<VolumeDim>,
+    return TestHelpers::test_creation<
```

Comment:
Could you add some upgrade instructions to the PR description please, to tell people about the changes to the `test_creation` functions?

Reply:
Those won't be published until May, which will probably be way too late to help anyone.  A post to Slack or something would be good.

There should probably be upgrade instructions about the metavariables changes, though.

Reply:
Yea we should probably render the current upgrade instructions to spectre-code.org